interfaces/Portalicious/src/app/pages/projects-overview/project-summary-card/project-summary-card.component.html

```diff
@@ -0,0 +1,110 @@
+<ng-template
+  #metric
+  let-value="value"
+  let-label="label"
+>
+  <dd class="px-2 py-1 font-bold">
+    @switch (metrics.status()) {
+      @case ('pending') {
+        <p-skeleton
+          class="my-1 cursor-progress text-lg"
+          height="1.125rem"
+          [width]="randomWidth"
+        />
+      }
+      @case ('success') {
+        {{ value }}
+      }
+    }
+  </dd>
+  <dt class="px-2 py-1 text-sm">
+    {{ label }}
+  </dt>
+</ng-template>
+
+<p-card
+  styleClass="group relative h-full border border-transparent *:h-full focus-within:border-primary hover:border-primary [&_.p-card-title]:flex-grow [&_.p-card-content]:flex-end"
+>
+  <ng-template pTemplate="title">
+    <a
+      [routerLink]="['/', AppRoutes.project, id()]"
+      class="after:absolute after:inset-0 after:block group-focus-within:text-primary group-hover:text-primary"
+    >
+      @switch (project.status()) {
+        @case ('pending') {
+          <p-skeleton
+            class="my-1 cursor-progress text-xl"
+            height="1.25rem"
+            [width]="randomWidth"
+          />
+        }
+        @case ('success') {
+          <span
+            class="line-clamp-2 text-ellipsis"
+            [title]="project.data()?.titlePortal | translatableString"
+          >
+            {{ project.data()?.titlePortal | translatableString }}
+          </span>
+        }
```

Comment:
This size is 'somewhat' the same size as the "loaded `titlePortal`"; But it took quite some manual fiddling.

There doesn't seem to be a way to style/size the skeleton with "font-size classes" or something; so that we could style the outer container and have only the skeleton OR the final-value in it. (Without any jumpiness in the page)